lib/bootstrap_form/form_builder.rb

```diff
@@ -314,6 +325,7 @@ def required_attribute?(obj, attribute)
     def form_group_builder(method, options, html_options = nil)
       options.symbolize_keys!
       html_options.symbolize_keys! if html_options
+      options[:id] = html_options[:id] if html_options && @options[:skip_default_ids]
```

Comment:
Could you explain why this line is needed?

Reply:
I'm pretty sure this is part my attempt to handle what is more elegantly handled in #357. The (minimized) tests pass without this line, so I'll take it out.